### DISCUSSION TITLE
🥇🥇🥇🥇 GitHub Solutions to Issues UI 🥇🥇🥇🥇

### DISCUSSION THREAD

**Discussion by bayurzxsmtp** [community/community#11161] [Projects and Issues]
I think it would be efficient if GitHub creates a special part in the UI where the best solution to an issue is displayed, instead of users wasting time scrolling through thousands of comments to find an answer!.

If you think I am right make sure to add a comment to gain attention!

**Comment by suprstarrd** [comment]
The "if you think I am right add a comment" feels kind of like cheating *but* I do usually do that anyways so I'll do it. Bump!

This could be done in a similar way to discussions. Definitely would be helpful.
